charts/hono/README.md

```diff
@@ -536,14 +536,17 @@ of Quarkus based services images.
 The chart can be configured to use Kafka as the messaging network instead of an AMQP 1.0 messaging network.
 The property `messagingNetworkType` is used to select the type of the messaging network.
 
-The following command provides a quickstart for Kafka based messaging (ensure `minikube tunnel` is running when using Minikube):
+The following command provides a quickstart for Kafka based messaging when using Minikube (ensure `minikube tunnel` is running ):
```

Comment:
Is this change needed? There is no minikube requirement here, or is there?

Reply:
This does not only work on Minikube but it won't work for example on microk8s out of the box. _If_ you use Minikube, as it assumed for most of the examples, then you can use this command, if not, you might have to use another configuration. Before it sounded like you only have special requirements on Minikube but on other Kubernetes distributions it would just work without additional steps.

Reply:
It's just that I think that people wanting to deploy Hono on some cloud-based Kubernetes cluster will not feel addressed here when explicitly only mentioning minikube. Therefore I would rather keep this generic and maybe add a note regarding microk8s or other distributions on what to take care of regarding the configuration.

Reply:
I'll just revert this line. The hint about using it without a load balancer comes two sentences later anyway.